Installation/cmake/modules/CGAL_NanoSVG_support.cmake

```diff
@@ -0,0 +1,6 @@
+if(NanoSVG_FOUND AND NOT TARGET CGAL::NanoSVG_support)
```

Comment:
During configuration I had to `lib/cmake/NanoSVG`   inside the build directory for the Nanosvg_dir  instead of just giving the build directory.